### PR TITLE
fix: add missing binning value cuboid bounds

### PR DESCRIPTION
**Pull request by asalzburger**
The `CuboidVolumeBounds` were missing an overload for the method:

```c++
  double binningBorder(BinningValue bValue) const final;
```

which makes the `Acts::TrackingVolumeArrayCreator` create garbage binning.

This PR fixes this.

### CODE REVIEW

File: Core/include/Acts/Geometry/CuboidVolumeBounds.hpp [hunk 1]
```diff
@@ -122,6 +122,13 @@ class CuboidVolumeBounds : public VolumeBounds {
                                   const Vector3& envelope = {0, 0, 0},
                                   const Volume* entity = nullptr) const final;
 
+  /// Binning borders in double
+  ///
+  /// @param bValue is the binning schema used
+  ///
+  /// @return float offset to be used for the binning
+  double binningBorder(BinningValue bValue) const final;
```
Comment (paulgessinger) [inline Core/include/Acts/Geometry/CuboidVolumeBounds.hpp:130]: So this was using the default from the base class?

**Review by paulgessinger — APPROVED** [review]
Ok

File: Core/include/Acts/Geometry/CuboidVolumeBounds.hpp (hunk 1 above)
Comment (asalzburger) [reply Core/include/Acts/Geometry/CuboidVolumeBounds.hpp:130]: yes, bad.

**Comment by acts-project-service** [comment]
## :red_circle: Athena integration test results
### Build job with this PR failed!
Please investigate the [build job](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/32123198) for the [pipeline](https://gitlab.cern.ch/acts/acts-athena-ci/-/pipelines/6131183)!

**Pull request merged**